pyproject.toml

```
@@ -9,10 +9,10 @@ readme = "README.md"
 license.file = "LICENSE"
 dynamic = ["version"]
 authors = [
-    { name = "S. Junges", email = "sjunges@cs.ru.nl" },
+    { name = "S. Junges", email = "sebastian.junges@ru.nl" },
     { name = "M. Volk", email = "m.volk@tue.nl" }
 ]
-requires-python = ">=3.7"
+requires-python = ">=3.11" # Can be reduced again, but want to iterate quicker through tests for building wheels.
```

Should be reduced again.
```suggestion
requires-python = ">=3.9"
```